intents/util/aspects.py

```diff
@@ -0,0 +1,192 @@
+"""This module implements the aspects"""
```

Comment:
"This module implements the aspects" is not a descriptive file comment.

Please use some words to explain what me mean by "aspects".